stm32-data-gen/src/chips.rs

```diff
@@ -468,6 +468,36 @@ impl PeriMatcher {
             ("STM32L5.*:TAMP:.*", ("tamp", "l5", "TAMP")),
             ("STM32U5.*:TAMP:.*", ("tamp", "u5", "TAMP")),
             ("STM32WL.*:TAMP:.*", ("tamp", "wl", "TAMP")),
+            (
+                "STM32L4R[59].*:OCTOSPIM:OCTOSPIM:octospi_v1_0.*",
+                ("octospim", "v1", "OCTOSPIM"),
+            ),
+            (
+                "STM32U5.*:OCTOSPIM:OCTOSPIM:octospi1_v3_0.*",
+                ("octospim", "v2", "OCTOSPIM"),
+            ),
+            (".*:OCTOSPIM:.*", ("octospim", "v2", "OCTOSPIM")), //Next chips releasing OCTOSPIM will default to v2.
```

Comment:
please don't add blanket matches like this. We don't know what next chips will do, it's better to not match registers and let the first person that needs to use octospi to check whether the regsiters are actually unchanged and add a new match rule.

Reply:
Makes perfect sense. I think I misinterpreted other blanket matches (which defaults to v1, e.g. `EXTI`, but in hindsight, I can see they server a different purpose.